cyphernodeconf_docker/templates/gatekeeper/api.properties

```diff
@@ -72,6 +72,7 @@ action_removefrombatch=spender
 action_listbatchers=spender
 action_getbatcher=spender
 action_getbatchdetails=spender
+action_generatetoaddress=spender
```

Comment:
It's actually `bitcoin_generatetoaddress`
This doesn't work when request goes through the gatekeeper.
Maybe we should add going through the gatekeeper in all the tests